### DISCUSSION TITLE
Ability to "Lock" settings, specifically "Macros"

### DISCUSSION THREAD

**Discussion by Chipperchap** [WarmUpTill/SceneSwitcher#1249] [General]
We have many users use our PC and occasionally we have to reset Advanced Scene Switcher the macro's because someone has changed them due to not knowing what they are doing. Please, is there some way we can lock the settings / macro section to prevent curious clickers?  Or is this more of a question for OBS?

**Comment by WarmUpTill** [comment]
>Please, is there some way we can lock the settings / macro section to prevent curious clickers?

Unfortunately, at the moment no such option exists.

>Or is this more of a question for OBS?

I don't think so.

Just an alternative idea:
Maybe you can [export / import the settings](https://github.com/WarmUpTill/SceneSwitcher/wiki/Saving-and-loading-settings#how-to-create-and-import-a-backup-of-your-settings) to save the working setup and revert any unwanted changes?

**Reply by Chipperchap** [reply]
> > Please, is there some way we can lock the settings / macro section to prevent curious clickers?
> 
> Unfortunately, at the moment no such option exists.

Okay - thank you so much. 
> 
> > Or is this more of a question for OBS?
> 
> I don't think so.

Thank you for that. 

> 
> Just an alternative idea: Maybe you can [export / import the settings](https://github.com/WarmUpTill/SceneSwitcher/wiki/Saving-and-loading-settings#how-to-create-and-import-a-backup-of-your-settings) to save the working setup and revert any unwanted changes?

Awesome. I had forgotten about that. 

In the meantime I have "removed" the A.S.Switcher "Dock" so that it is unseen by others. Hopefully that is enough to stop those who interfere as most don't know how the "docks" feature works. 😊

**Comment by Chipperchap** [comment]
Hi @WarmUpTill - is there some way I can make a donation toward your efforts / project please?

**Reply by WarmUpTill** [reply]
Thank you very much for the offer! 🙂

There is no way to donate to me or the project.
Please donate to a charity or the OBS project instead.
